routes/api/contacts.js

```diff
@@ -46,8 +46,8 @@ router.post("/api/contacts", async (req, res, next) => {
 
 router.delete("/api/contacts/:id", async (req, res, next) => {
   try {
-    await contacts.removeContact(req.params.id);
-    res.json({ message: "Contact deleted" });
+    const deleteContact = await contacts.removeContact(req.params.id);
+    res.json(deleteContact);
   } catch (error) {
     res.status(404).json({ message: "Not found" });
```

Comment:
Dlatego że tam `contacts.removeContact` jest dodany throw w bloku catch to jeżeli pojawią się błędy to tutaj zostaną przechwycone. :)